Library/Homebrew/utils/lock.sh

```diff
@@ -0,0 +1,34 @@
+# The file descriptor used in the lock.
+# This file descriptor should be used exclusively for lock purpose.
+# Any value except 0(stdin), 1(stdout) and 2(stderr) can do the job.
```

Comment:
FD is unique per process. It is only shared to subprocess. So I don't think there is a problem here.